src/routing.jl

```diff
@@ -44,6 +44,23 @@ page(p::AbstractString, app...) = page(splitpath(p), app...)
 page(app...) = page([], app...)
 page(app::Function, p) = page(p, app)
 
+# Query routing
+
+function matchquery(q, req)
+  qdict = parsequerystring(req[:query])
```

Comment:
I'm trying to see if I can rebase this PR on top of master, but this one call `parsequerystring` doesn't exist, and I'm having trouble locating it in the git history. Does anyone know what this function is supposed to do? @sarvjeetsinghghotra @MikeInnes

Reply:
https://github.com/JuliaWeb/HttpCommon.jl/search?q=parsequerystring&unscoped_q=parsequerystring